tests/suites/hup/suite.js

```diff
@@ -334,6 +334,7 @@ module.exports = {
 						persistentLogging: true,
 						// Set local mode so we can perform local pushes of containers to the DUT
 						localMode: true,
+						apiEndpoint: 'https://api.balena-cloud.com',
```

Comment:
```suggestion
						apiEndpoint: `https://api.${this.suite.options.balena.apiUrl}`,
```

Making it configurable. Also, should we be adding this to OS configs in rest of the tests too?

Reply:
yeah in the rest of the test suites we are doing it

Reply:
Why is the `api.` prefix not part of the `apiUrl` setting?

Reply:
I think it should be, historically we have never supported any other API endpoint other production or staging balenaCloud. Adding this into my https://github.com/balena-os/leviathan/issues/926